lib/models/userscript_model.dart

```diff
@@ -22,20 +22,20 @@ enum UserScriptUpdateStatus {
 }
 
 class UserScriptModel {
-  UserScriptModel({
```

Comment:
There's something with the autoformater here. Even `dart format` does this and I have zero idea why.